rts/motoko-rts/src/gc/copying.rs

```diff
@@ -23,7 +23,7 @@ unsafe fn copying_gc<M: Memory>(mem: &mut M) {
 
     copying_gc_internal(
         mem,
-        ic::get_heap_base(),
+        ic::get_aligned_heap_base(),
```

Comment:
What is the motivation for this change?

Reply:
It is actually unrelated to this PR but only a unification of the heap alignment, mentioned in: 
https://github.com/dfinity/motoko/pull/3829#discussion_r1119110197
Not sure whether I should revert this to keep the PR more coherent?